test/functional/isVisible.html

```diff
@@ -2,86 +2,54 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <link rel="shortcut icon" href="">
```

Comment:
What's this <link> for, anyway?

Reply:
Firefox seems to, by default, make a GET request to get the page's favicon for display in the tab. If I don't have a favicon, my HTTP server throws an error.

This is an empty favicon to prevent that error. I can make a comment saying as much.

Reply:
gtk! A comment would be great for Future Me.